src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -106,6 +106,24 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
     hasEmailSettingsPanelExpanded: false,
   };
 
+  // to get the original session model on discard changes
+  feedbackSessionModel: FeedbackSession = {
```

Comment:
Maybe name it as `feedbackSessionModelBeforeEditing`?

Reply:
I agree, `feedbackSessionModelBeforeEditing` is more descriptive. Changing now.